src/plots/cartesian/set_convert.js

```diff
@@ -486,6 +549,51 @@ module.exports = function setConvert(ax, fullLayout) {
             ax._b = -ax._m * rl0;
         }
 
+        // set of "N" disjoint breaks inside the range
+        ax._breaks = [];
+        // length of these breaks in value space
+        ax._lBreaks = 0;
+        // l2p slope (same for all intervals)
+        ax._m2 = 0;
+        // set of l2p offsets (one for each of the (N+1) piecewise intervals)
+        ax._B = [];
```

Comment:
I am afraid using `_B` here may be too abstract. 
How about `_offBr` or other alternatives?

Reply:
I can change it if you want, but `ax._B` refers to `ax._b` which is used on axes w/o breaks.

Reply:
Then adding a comment may be enough.